lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1161,6 +1160,22 @@ parseAnnotations(StringRef AnnotatedCode) {
 void addIntelFPGADecorations(
     SPIRVEntry *E,
     std::vector<std::pair<Decoration, std::string>> &Decorations) {
+  // FIXME: move this code out of this function, it is not IntelFPGA-specific
```

Comment:
Would this be easily resolved now?

Reply:
I've refactored code related to IntelFPGA-specific decorations and removed this FIXME